mephisto/abstractions/providers/prolific/prolific_provider.py

```diff
@@ -0,0 +1,366 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+from dataclasses import dataclass
+from dataclasses import field
+from datetime import datetime
+from datetime import timezone
+from typing import Any
+from typing import cast
+from typing import ClassVar
+from typing import List
+from typing import Type
+from typing import TYPE_CHECKING
+
+from mephisto.abstractions.crowd_provider import CrowdProvider
+from mephisto.abstractions.crowd_provider import ProviderArgs
+from mephisto.abstractions.providers.prolific import prolific_utils
+from mephisto.abstractions.providers.prolific.api.constants import ProlificIDOption
+from mephisto.abstractions.providers.prolific.prolific_agent import ProlificAgent
+from mephisto.abstractions.providers.prolific.prolific_datastore import ProlificDatastore
+from mephisto.abstractions.providers.prolific.prolific_requester import ProlificRequester
+from mephisto.abstractions.providers.prolific.prolific_unit import ProlificUnit
+from mephisto.abstractions.providers.prolific.prolific_worker import ProlificWorker
+from mephisto.abstractions.providers.prolific.provider_type import PROVIDER_TYPE
+from mephisto.operations.registry import register_mephisto_abstraction
+from mephisto.utils.logger_core import get_logger
+from mephisto.utils.qualifications import QualificationType
+from mephisto.utils.qualifications import worker_is_qualified
+from .api.client import ProlificClient
+from .api.data_models import ParticipantGroup
+from .api.data_models import Project
+from .api.data_models import Study
+from .api.data_models import Workspace
+from .api.eligibility_requirement_classes.participant_group_eligibility_requirement import (
+    PARTICIPANT_GROUP_ELIGIBILITY_REQUIREMENT
+)
+from .api.exceptions import ProlificException
+
+if TYPE_CHECKING:
+    from mephisto.data_model.task_run import TaskRun
+    from mephisto.data_model.unit import Unit
+    from mephisto.data_model.worker import Worker
+    from mephisto.data_model.requester import Requester
+    from mephisto.data_model.agent import Agent
+    from mephisto.abstractions.blueprint import SharedTaskState
+    from omegaconf import DictConfig
+
+
+DEFAULT_FRAME_HEIGHT = 0
+DEFAULT_PROLIFIC_GROUP_NAME_ALLOW_LIST = "Allow list"
+DEFAULT_PROLIFIC_GROUP_NAME_BLOCK_LIST = "Block list"
+DEFAULT_PROLIFIC_PROJECT_NAME = "Project"
+DEFAULT_PROLIFIC_WORKSPACE_NAME = "My Workspace"
+
+logger = get_logger(name=__name__)
+
+
+@dataclass
+class ProlificProviderArgs(ProviderArgs):
+    """Base class for arguments to configure Crowd Providers"""
+
+    _provider_type: str = PROVIDER_TYPE
+    requester_name: str = PROVIDER_TYPE
```

Comment:
This should be `OmegaConf.MISSING` or None.

Reply:
`requester_name` already defaults missing here: https://github.com/facebookresearch/Mephisto/blob/25c2594be19100abd40037bf9bb43d8a3a87c6ca/mephisto/abstractions/crowd_provider.py#L25
I think that's what's making the error get thrown, though for a wrong reason.